### DISCUSSION TITLE
Getting file links to work with new jetbrains:// toolbox links.

### DISCUSSION THREAD

**Discussion by skeemer** [spatie/ray#838] [General]
The new links look like 
jetbrains://php-storm/navigate/reference?project=project-name&path=app/Livewire/Admin/ComponentChild.php:110

My config ray.config includes:
```
'remote_path' => env('RAY_REMOTE_PATH', '/var/www/html/'),
'local_path' => env('RAY_LOCAL_PATH', 'project=project-name&path='),
```

Ray -> Preferences -> Custom URL
```
jetbrains://php-storm/navigate/reference?%path:%line
```

This mostly works, except it appears that the %path is getting escaped when substituted. PHPStorm is seeing:
```
jetbrains://php-storm/navigate/reference?project%3Dproject-name%26path%3Dapp%2FLivewire%2FAdmin%2FComponentChild.php:110
```

Is this a bug or am I missing another option somewhere?

**Comment by skeemer** [comment]
BTW, I'm trying to get this to work as Jetbrains Toolbox supports this URL out of the box. No registry hacks or anything like that needed.

**Comment by hrsa** [comment]
Yep, i'm trying to make it work too - and i have the same problem with the escaped %path.
We probably also need %project variable ?
